player/lua/console.lua

```diff
@@ -116,6 +107,28 @@ local default_item
 local set_active
 
 
+local function get_font()
+    if opts.font ~= '' then
+        return opts.font
+    end
+
+    if selectable_items and not searching_history then
+        return
+    end
+
+    -- Pick a better default font for Windows and macOS
+    if platform == 'windows' then
+        return 'Consolas'
```

Comment:
Could we switch to modern font? Nothing is using `Consolas` anymore. `Cascadia Mono` should be used with a fallback to `Consolas` or `monospace` if not available. It is built-in Windows 11 and on Windows 10 it is likely to be also available, but not on clean install.

Reply:
Specifying multiple fonts doesn't seem to work: `show-text '${osd-ass-cc/0}{\fnmonospace}{\fnnon_existing}foo'` uses `--osd-font`. We can just let Cascadia Mono fallback to `--osd-font` if you want, the console is still usable, that's how I always used it.

Reply:
Yes, that would work.

Reply:
@guidocella: There is space missing in commit description, if you want to fix that. Also I will think about it, because I don't want to make Windows 10 IoT never update users to complain that font changed to osd default one.

Reply:
@na-na-hi: Do you have opinion on that? Would it be acceptable to fallback to osd-font for Windows 10 users? I think various products may install Cascadia Code, but it may not be available on clean install.

Reply:
+1 for Consolas as default on Windows. It's more common, and still conservatively nice.

Reply:
Ok, fair. To be honest I was expecting we can specify fallback fonts in same as one can in CSS, I agree that Cascadia is likely not available on Windows 10 and this is still big target. We can reevaluate in few years. @guidocella would you drop this commit for now? Thanks.